src/Mobs/Monster.cpp

```diff
@@ -95,12 +95,16 @@ cMonster::cMonster(const AString & a_ConfigName, eMonsterType a_MobType, const A
 	, m_DropChanceBoots(0.085f)
 	, m_CanPickUpLoot(true)
 	, m_BurnsInDaylight(false)
-	, m_RelativeWalkSpeed(1.0)
+	, m_RelativeWalkSpeed(0.5)
```

Comment:
Why should the default be 1?  Sometimes the momentum causes mobs to fall off cliffs and miss their pathfinding waypoints.

Can we keep it 0.5 for now? The AI needs some more thorough revision later, but for now, this is the fastest fix.